src/Compilers/VisualBasic/Test/Semantic/Diagnostics/GetDiagnosticsTests.vb

```diff
@@ -674,5 +674,60 @@ End Class"
                 reportDiagnostic(CodeAnalysis.Diagnostic.Create(Descriptor, location, containingSymbol.Name))
             End Sub
         End Class
+
+        <Fact, WorkItem("https://github.com/dotnet/roslyn/issues/68654")>
+        Public Async Function TestAnalyzerLocalDiagnosticsWhenReportedOnFieldSymbol() As Task
```

Comment:
>TestAnalyzerLocalDiagnosticsWhenReportedOnFieldSymbol [](http://example.com/codeflow?start=30&length=53)

Is it possible to add similar test for C#?

Reply:
Yes, definitely. Will do.